pkg/aws/ec2/api/eni_cleanup_test.go

```diff
@@ -85,11 +85,21 @@ func TestENICleaner_cleanUpAvailableENIs(t *testing.T) {
 
 	gomock.InOrder(
 		// Return network interface 1 and 2 in first cycle
-		mockWrapper.EXPECT().DescribeNetworkInterfaces(mockDescribeNetworkInterfaceIp).
-			Return(mockDescribeInterfaceOpWith1And2, nil),
+		mockWrapper.EXPECT().DescribeNetworkInterfacesPages(gomock.Any(), gomock.Any()).
```

Comment:
Can we use the expected sample input here instead of any input?

Reply:
I think we should be fine to use Any() in this case because we invoke DoAndReturn() which declares the action and will execute the mocked function and the args will be checked. Please let me know if you think otherwise. Thanks.